src/contexts/theme-context/theme-context.js

```diff
@@ -0,0 +1,28 @@
+import { createContext, useContext, useState, useEffect } from "react";
+
+const ThemeContext = createContext("");
+
+const ThemeProvider = ({ children }) => {
+  const [theme, setTheme] = useState(
+    JSON.parse(localStorage.getItem("userTheme")) ?? false
```

Comment:
reading a local storage value is a side effect, so read it in useEffect and  then set the state inside that

Reply:
Done, Thank you